Makefile

```
@@ -115,7 +115,8 @@ slow: fast run-notebooks test-examples docker
 
 docker:
 	docker build -f docker/Dockerfile -t $(DOCKER_IMAGE) .
-	docker run -ti --rm $(DOCKER_IMAGE) build --fov-count 1 --primary-image-dimensions '{"z": 1}' /tmp/
+	starfish --help
```

I think you would want to run this in the docker container.

```suggestion
docker run -ti --rm spacetx/starfish -- --help
```

---

yuuuuuup my b